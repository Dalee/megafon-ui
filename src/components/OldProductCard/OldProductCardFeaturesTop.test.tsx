import * as React from 'react';
import { shallow } from 'enzyme';
import OldProductCardFeaturesTop from './OldProductCardFeaturesTop';
import Checked from 'icons/checked_24.svg';

const props = {
    title: 'test',
    params: [
        {
            svgIcon: <Checked />,
            title: 'test',
            caption: 'test',
            value: 'test',
        },
    ],
    socialIcons: [{
        svgIcon: <Checked />,
        title: 'test',
    }],
};

describe('<OldProductCardFeaturesTop />', () => {
    it('it renders OldProductCardFeaturesTop', () => {
        const wrapper = shallow(<OldProductCardFeaturesTop {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
