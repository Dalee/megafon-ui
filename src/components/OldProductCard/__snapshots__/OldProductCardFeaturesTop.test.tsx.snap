// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OldProductCardFeaturesTop /> it renders OldProductCardFeaturesTop 1`] = `
<div
  className="old-product-card-features-top"
>
  <div
    className="old-product-card-features-top__inner"
    key="test0"
  >
    <Header
      as="h3"
      className="old-product-card-features-top__title"
      color="black"
      margin={false}
    >
      test
    </Header>
    <div
      className="old-product-card-features-top__text"
    >
      test
    </div>
    <DropdownSocialList
      icons={
        Array [
          Object {
            "svgIcon": <test-file-stub />,
            "title": "test",
          },
        ]
      }
      maxIconNumber={6}
    />
  </div>
</div>
`;
